shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/FirebaseAuthenticationApi.kt

```diff
@@ -21,29 +23,41 @@ class FirebaseAuthenticationStore : AuthenticationStore {
         email: String,
         password: String,
         returnSecureToken: Boolean
-    ) = handleAuthenticationRequest(apiKey,
-        ENDPOINT_SIGN_UP, email, password, returnSecureToken)
+    ) = handleAuthenticationRequest(
+        apiKey,
+        ENDPOINT_SIGN_UP, email, password, returnSecureToken
```

Comment:
minor again: can we have them all on one line or all on separate lines please?

Reply:
Done